index.bs

```diff
@@ -451,6 +462,7 @@ dictionary XRSessionCreationOptions {
   XRPresentationContext? outputContext = null;
 };
 </pre>
+</section>
```

Comment:
It's probably a silly nit... but I'd kinda like the box not to span across top level headings...

Reply:
It's a fair point! I've updated this for everything *but* the privacy and security section, because that feels like it should all be taken as a whole anyway.